readthedocs/projects/views/private.py

```diff
@@ -622,9 +622,7 @@ def post(self, request, *args, **kwargs):
             username=username,
         )
         if self._is_last_user():
-            return HttpResponseBadRequest(
-                _(f"{username} is the last owner, can't be removed")
-            )
+            return HttpResponseBadRequest(_("User is the last owner, can't be removed"))
```

Comment:
It would be good to write a comment here explaining why we removed `{username}` from here so we don't do it again.